tests/unit-tests/formatting/functions.php

```diff
@@ -685,7 +685,11 @@ public function test_wc_timezone_string() {
 
 		// Test with manually set UTC offset.
 		update_option( 'gmt_offset', -4 );
-		$this->assertNotEquals( 'UTC', wc_timezone_string() );
+		$this->assertEquals( 'UTC', wc_timezone_string() );
+
+		// Test with float GMT offset.
+		update_option( 'gmt_offset', 2.5 );
+		$this->assertEquals( 'UTC', wc_timezone_string() );
```

Comment:
I also believe that this test is incorrect. I think that instead of `UTC`, `wc_timezone_string()` should return the string for the first `timezone_id` that is returned by `timezone_abbreviations_list()` for the given timezone offset, no?